### DISCUSSION TITLE
Удаление дубликатов (буквы в разном регистре)

### DISCUSSION THREAD

**Discussion by KisoidPoh095** [Chimildic/YaMuTools#19] [Помощь с ошибкой] [answered]
Не распознались как дубликаты:
The Shadow **o**f Your Smile https://music.yandex.ru/album/14668/track/151377
The Shadow **O**f Your Smile https://music.yandex.ru/album/46319/track/447695

Наверное, имеет смысл привести строки к единому регистру, прежде чем сравнивать.
У меня в PowerShell это делается так: $somestring.ToUpper()
Не знаю, как в JavaScript...

**Comment by Chimildic** [comment, marked as answer]
Добавлено дополнительное форматирование при сравнении дубликатов с версии 0.7.6

Обратите внимание, что треки с одинаковыми названиями не удаляются, а перемещаются наверх плейлиста для пользовательской модерации. Удаляются сразу только дубли по _id_.

**Reply by KisoidPoh095** [reply]
Да, теперь эти два трека "всплыли" наверх. Проверял под Firefox.
